lib/main.coffee

```diff
@@ -9,71 +8,49 @@ module.exports =
       type: 'string'
       default: 'foodcritic'
     extraArgs:
+      title: 'Extra Arguments'
+      description: 'Extra arguments to pass to foodcritic'
       type: 'string'
       default: ''
 
   activate: ->
     @subscriptions = new CompositeDisposable
-    @MessageRegexp = null
-    @subscriptions.add atom.config.observe 'foodcritic.executablePath', =>
-      @executablePath = atom.config.get 'linter-foodcritic.executablePath'
-    @subscriptions.add atom.config.observe 'foodcritic.extraArgs', =>
-      @extraArgs = atom.config.get 'linter-foodcritic.extraArgs'
+    @MessageRegexp = /(FC\d{3}:\s.+):(?:.+):(\d+)/g
```

Comment:
It would make it easier for the eyes if you would move this variable into the `provideLinter` function, you would also not have to use the `@` then. It's not important though